helmchart/sdp/values.yaml

```diff
@@ -283,3 +281,47 @@ anchorPlatform:
   # - SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET
   # - SECRET_SEP24_MORE_INFO_URL_JWT_SECRET
   # - SECRET_PLATFORM_API_AUTH_SECRET
+
+
+# =========================== FRONTEND DASHBOARD ===========================
+dashboard:
+  enabled: false
+  route:
+    schema: "https"
+    domain: "sdp-dashboard.localhost.com"
+    port: "80"
+  image:
+    fullName: stellar/stellar-disbursement-platform-frontend:latest
+    pullPolicy: IfNotPresent
+  deployment:
+    annotations:
+      fluxcd.io/ignore: "true"
```

Comment:
💡  `fluxcd` annotations are configs that are specific to SDF ops infrastructure (they may not be used by other orgs/companies). Should we omit them from here and just add them in our kube repo ?